src/main/java/dev/andrylat/vomelianchuk/CardNumberOperations/CardNumberProcess.java

```diff
@@ -0,0 +1,118 @@
+package dev.andrylat.vomelianchuk.CardNumberOperations;
+
+import dev.andrylat.vomelianchuk.CardNumberOperations.enums.Errors;
+import dev.andrylat.vomelianchuk.CardNumberOperations.enums.PaymentSystem;
+
+import java.util.Optional;
+
+public class CardNumberProcess {
+    private static final int LENGTH_CARD_NUMBER = 16;
+    private static final String HIATUS_SEPARATOR = "\\s";
+    private static final int TEN = 10;
+
+    private String cardNumber;
+    private PaymentSystem paymentSystem;
+    private Errors error;
```

Comment:
Only one error can be ? There are few errors for the same input in the example.

Reply:
Fixed here: 1783555